### DISCUSSION TITLE
Welcome to htsc Discussions!

### DISCUSSION THREAD

**Discussion by samlinux** [samlinux/htsc#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sinjab** [comment]
Thank you Roland. I deeply appreciate your efforts.

**Comment by sinjab** [comment]
I have a question about hyperledger fabric profile SampleDevModeSolo. This profile is very useful for debugging. How to construct the connection.yaml for such profile? I want to use it to connect a client to the contracts.

**Reply by samlinux** [reply]
I think you can do it in the same way as you would do it for a normal network. Check out the test-network setup. There you can find a script ccp-generate.sh https://github.com/hyperledger/fabric-samples/blob/master/test-network/organizations/ccp-generate.sh

**Reply by sinjab** [reply]
There is an important difference with this SampleDevModeSolo profile. There is no certificate authority server. So how to generate an identity and a wallet for the client.

**Reply by samlinux** [reply]
ok, give me some time then I will look into that.

**Reply by samlinux** [reply]
you are going to try to connect with the node.js SDK to the test-network right?

**Reply by sinjab** [reply]
Thank you. Yes node.js SDK. But not the test-network. Launching one orderer and one peer from the  command line from the binaries with SampleDevModeSolo profile. One organization, no ca, no docker.

**Reply by sinjab** [reply]
I’m following this:
https://github.com/samlinux/htsc/blob/master/meetup-290121/fabric2DevMode.md

**Reply by samlinux** [reply]
that's good, then we can extend this guide, but I think there should no difference between docker and binary version.
